run_unittests.py

```diff
@@ -5665,6 +5665,21 @@ def test_pkgconfig_link_order(self):
         deps = stdo.split()
         self.assertTrue(deps.index(b'-lsomething') < deps.index(b'-ldependency'))
 
+    @skipIfNoPkgconfig
+    def test_pkgconfig_cross_linkdir(self):
+        '''
+        Test that libraries are listed before their dependencies.
+        '''
+        testdir = os.path.join(self.unit_test_dir, '73 cross pkgconfig libdir')
+        self.init(testdir, extra_args=['-Dgenerate=true'])
+        self.meson_cross_file = os.path.join(self.builddir, "crossfile")
+        self.assertTrue (os.path.exists(self.meson_cross_file))
+        self.new_builddir()
+        self.init(testdir, default_args=False,
+                 extra_args=['--cross-file', self.meson_cross_file])
```

Comment:
__[Flake8]__ continuation line under-indented for visual indent ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6461#issue-5064099))

<details>

| Rule |
| - |
| E128 |

</details>
<sub>posted by Sider</sub>